ipaserver/install/ipa_server_certinstall.py

```diff
@@ -134,14 +134,18 @@ def install_http_cert(self):
         dirname = certs.NSS_DIR
 
         old_cert = installutils.get_directive(paths.HTTPD_NSS_CONF,
-                                              'NSSNickname')
+                                              'NSSNickname', quote_char="'")
```

Comment:
You are right. I think that I will delegate the quoting/unquoting transformations to separate functions. That will be more robust IMHO.